tensorflow-app-starters-test-support/src/main/java/org/springframework/cloud/stream/app/test/tensorflow/TensorflowProcessorTestConfiguration.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 the original author or authors.
```

Comment:
My first attempt to use the intellij copyright auto-update was mess :) I hope i fixed the configuration now 
✅